client/a8c-for-agencies/sections/overview/overview.tsx

```diff
@@ -6,6 +6,9 @@ import LayoutHeader, {
 } from 'calypso/a8c-for-agencies/components/layout/header';
 import LayoutTop from 'calypso/a8c-for-agencies/components/layout/top';
 import MobileSidebarNavigation from 'calypso/a8c-for-agencies/components/sidebar/mobile-sidebar-navigation';
+import ContentSidebar from 'calypso/jetpack-cloud/components/content-sidebar';
```

Comment:
I think this import should be:

```
import ContentSidebar from 'calypso/a8c-for-agencies/components/content-sidebar';
```

Will fix a duplicate mobile menu from being rendered. 

(Couldn't inline suggest this as Prettier will want to move the import higher up after changing it.)

Reply:
Good catch. Thank you!